source/common/router/config_utility.cc

```diff
@@ -45,5 +45,23 @@ bool ConfigUtility::matchHeaders(const Http::HeaderMap& request_headers,
   return matches;
 }
 
+Http::Code ConfigUtility::parseRedirectResponseCode(
```

Comment:
please make sure you have coverage over all of this. There should be some tests in config_impl_test.

Reply:
Added a test case in config_impl_test.cc

Reply:
You need an actual test that loads a route table, gets the route, and makes sure a code is set. You don't have any test to make sure that that part is working. This should be in config_impl_test also. This may require converting one of the tests over to v2 YAML like we have been doing lately in other tests.